project.clj

```diff
@@ -159,6 +159,7 @@
                  [org.clojure/core.async]
                  [org.clojure/core.match "0.3.0-alpha4"]
                  [org.clojure/core.memoize]
+                 [org.clojure/data.generators "1.0.0"]
```

Comment:
The version number seems to be required here. I have no idea why.

Reply:
For dependencies that are specified in [clj-parent](https://github.com/puppetlabs/clj-parent), we can omit the version and it will be inherited. This one isn't in clj-parent so it has to be specified explicitly.

The parent project is defined just above this section:
https://github.com/puppetlabs/puppetdb/pull/3792/files#diff-274071745a4e2a04b647d79d500537e6dc13eee54f44d0426140026293701d1bL149-L150

Reply:
Ah, so these are shared dependencies that can be optionally included across subprojects without each subproject fiddling version numbers requiring multiple prs to update.